cli/.goreleaser.yaml

```
@@ -32,9 +35,28 @@ dockers:
       - "ghcr.io/cloudquery/cloudquery:latest-linux-arm64"
       - "ghcr.io/cloudquery/cloudquery:{{.Version}}-linux-arm64"
       - "ghcr.io/cloudquery/cloudquery:{{ .Major }}.{{ .Minor }}-linux-arm64"
+    use: buildx
     build_flag_templates:
+      - "--pull"
+      - "--platform=linux/arm64"
       - "--label=org.opencontainers.image.source=https://github.com/cloudquery/cloudquery"
 
+docker_manifests:
+- name_template: "cloudquery/cloudquery:{{.Version}}"
+  image_templates:
+  - "cloudquery/cloudquery:{{.Version}}-linux-amd64"
+  - "cloudquery/cloudquery:{{.Version}}-linux-arm64"
+
+- name_template: "cloudquery/cloudquery:latest"
+  image_templates:
+  - "cloudquery/cloudquery:latest-linux-amd64"
+  - "cloudquery/cloudquery:latest-linux-arm64"
+
+- name_template: "cloudquery/cloudquery:{{ .Major }}.{{ .Minor }}"
+  image_templates:
+  - "cloudquery/cloudquery:{{ .Major }}.{{ .Minor }}-linux-amd64"
+  - "cloudquery/cloudquery:{{ .Major }}.{{ .Minor }}-linux-arm64"
```

```suggestion
- name_template: "cloudquery/cloudquery:{{.Version}}"
  image_templates:
  - "ghcr.io/cloudquery/cloudquery:{{.Version}}-linux-amd64"
  - "ghcr.io/cloudquery/cloudquery:{{.Version}}-linux-arm64"

- name_template: "ghcr.io/cloudquery/cloudquery:latest"
  image_templates:
  - "ghcr.io/cloudquery/cloudquery:latest-linux-amd64"
  - "ghcr.io/cloudquery/cloudquery:latest-linux-arm64"

- name_template: "ghcr.io/cloudquery/cloudquery:{{ .Major }}.{{ .Minor }}"
  image_templates:
  - "ghcr.io/cloudquery/cloudquery:{{ .Major }}.{{ .Minor }}-linux-amd64"
  - "ghcr.io/cloudquery/cloudquery:{{ .Major }}.{{ .Minor }}-linux-arm64"
```

And I think you need to remove:

```
      - "ghcr.io/cloudquery/cloudquery:latest"
      - "ghcr.io/cloudquery/cloudquery:{{.Version}}"
      - "ghcr.io/cloudquery/cloudquery:{{ .Major }}.{{ .Minor }}"
```

from the amd64 configuration.